src/cpp/r/R/Tools.R

```diff
@@ -499,13 +499,37 @@ assign(envir = .rs.Env, ".rs.hasVar", function(name)
    .rs.registerReplaceHook(name, package, unsupported)
 })
 
-.rs.addFunction( "setCRANRepos", function(reposUrl)
+.rs.addFunction( "isCRANReposList", function(repos) {
+  startsWith(repos, "CRAN|")
```

Comment:
`startsWith` is not available until R 3.3 (if memory serves we have a polyfill)

Reply:
Thanks!